ssqatest/tests/backend/misc_tests/test_create_new_product.py

```diff
@@ -0,0 +1,56 @@
+import pytest
+from ssqatest.src.api_helpers.ProductsAPIHelper import ProductsAPIHelper
+from ssqatest.src.dao.products_dao import ProductsDAO
+from ssqatest.src.utilities.genericUtilities import generate_random_string
+import logging as log
+pytestmark= [pytest.mark.backend]
+
+"""
+    This is a test to create a product using API post call and then asserting it was created.
+"""
+
+@pytest.mark.be001
+#create a payload and run a post API call to create a product
+def test_create_a_product():
+    payload=dict()
+    payload["name"]=generate_random_string(5,"lp")
+    payload["weight"]= "2"
+    payload["regular_price"]= "10.00"
+    # Make the API post call to create a product
+    product_create= ProductsAPIHelper().call_create_product(payload)
+    print(product_create)
+    product_id=product_create["id"]
+    product_name= product_create["name"]
+    product_wight= product_create["weight"]
+    product_price= product_create["regular_price"]
+    log.info("===================================")
+    log.info(f"The product id is: {product_id}")
+    log.info(f"The price is: ${product_price}")
+
+
+    # Verify that the product got created
+    assert product_id, f"The product {payload['name']}never got created! "
```

Comment:
missing space after between `}never` the print would come out weird. the product name and 'never' will be connected.